File: kom-frontend/src/app/domain/player.ts
import { Component } from "@angular/core";
import { Gender } from "./gender";

export interface Player {
    id: number;
    name: string;
    weightInKilogram: number;
    alcoholPercentage: number;
    drinkInCentiliter: number;
    totalAmountOfShots: number;
    widMark: number;
    gender: Gender;
}

File: kom-game/src/main/java/nl/rickverkuijlen/mormel/game/objects/Player.java
package nl.rickverkuijlen.mormel.game.objects;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class Player {
    private int id;
    private String name;
    private double weightInKilogram;
    private double alcoholPercentage;
    private double drinkInCentiliter;
    private double totalAmountOfShots;
    private double widMark;
    private Gender gender;
}
